src/Collection/Collection.service.ts

```diff
@@ -1,16 +1,129 @@
+import { HTTPError, STATUS_CODES } from '../common/HTTPError'
 import { collectionAPI } from '../ethereum/api/collection'
 import { isPublished } from '../utils/eth'
+import { isManager as isTPWManger } from '../ethereum/api/tpw'
+import { FactoryCollection } from '../ethereum/FactoryCollection'
+import { Ownable } from '../Ownable'
+import { CollectionAttributes, FullCollection } from './Collection.types'
+import { toDBCollection, decodeTPCollectionURN } from './utils'
+import { Collection } from './Collection.model'
 
+const DAY = 1000 * 60 * 60 * 24
 export class CollectionService {
   isLockActive(lock: Date | null) {
     if (!lock) {
       return false
     }
 
     const deadline = new Date(lock)
-    deadline.setDate(deadline.getDate() + 1)
+    return deadline.getTime() + DAY > Date.now()
```

Comment:
If we're not using `get/setDate()` anymore, we can just add `DAY` to the lock value:

```suggestion
    return lock + DAY > Date.now()
```